website/static/index.html

```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE html>
```

Comment:
why is this file deleted? whats with the rest of `static` folder?

Reply:
The static page caused errors after the update / the redirect did not work.

Has been replaced by adding Slug in website/docs/10-overview.md 
and making a redirect in
website/docusaurus.config.js

Not sure about the rest of the files in the Static folder tbh.